### PR TITLE
ASUIS-153: Department, profile and affiliation Mobile Mockups

### PR DESCRIPTION
**Pull request by travisbutterfield**
Department, profile and affiliation Mobile Mockups

### CODE REVIEW

File: web/themes/custom/webdir/templates/views/views-view--group-members--page-1.html.twig [hunk 1]
```diff
@@ -40,7 +40,8 @@
 <div{{ attributes.addClass(classes) }}>
   <div class="container">
     <div class="row">
-      <div class="col-12">
+<div class="col-12">
```
Comment (mlsamuelson) [inline web/themes/custom/webdir/templates/views/views-view--group-members--page-1.html.twig:43]: Indentation.

File: web/themes/custom/webdir/package-lock.json [hunk 2]
```diff
@@ -1,17760 +1,7 @@
...
-      "integrity": "sha1-PYyt2Q2XZWn6g1qx+OSyOhBWBac=",
-      "dev": true,
-      "engines": {
-        "node": ">= 0.6"
-      }
-    },
-    "node_modules/friendly-errors-webpack-plugin": {
-      "version": "1.7.0",
-      "resolved": "https://registry.npmjs.org/friendly-errors-webpack-plugin/-/friendly-errors-webpack-plugin-1.7.0.tgz",
-      "integrity": "sha512-K27M3VK30wVoOarP651zDmb93R9zF28usW4ocaK3mfQeIEI5BPht
```
Comment (mlsamuelson) [inline web/themes/custom/webdir/package-lock.json:4]: Can you walk me through the package-lock.json changes here? I see a lot of packages being removed. What accounts for this? Was there a lot of unused cruft you cleaned up?

File: composer.json [hunk 3]
```diff
@@ -148,6 +148,13 @@
     "sort-packages": true,
     "platform": {
       "php": "7.4"
+    },
+    "allow-plugins": {
+      "composer/installers": true,
+      "cweagans/composer-patches": true,
+      "drupal/core-composer-scaffold": true,
+      "oomphinc/composer-installers-extender": true,
+      "zaporylie/composer-drupal-optimizations": true
```
Comment (mlsamuelson) [inline composer.json:157]: Walk me through this. Are you adding it as a pro-active measure?

File: build-metadata.json [hunk 4]
```diff
@@ -0,0 +1 @@
+{"url":"git@github.com:ASUWebPlatforms/asu-isearch.git","ref":"master","sha":"fd7c35c77b82cebc0aa5a26d65ef81734a44429c","comment":"Merge pull request #112 from ASUWebPlatforms/hot-fix-migration","commit-date":"2022-02-25 05:43:04 -0700","build-date":"2022-02-25 12:45:14 +0000"}
```
Comment (mlsamuelson) [inline build-metadata.json:1]: I'm thinking this is a build artifact that's trying to sneak in. It says it was committed by Daniela, but the build runs under her deploy_key currently (something on my list to address), so I think it's from that. I saw something similar with a PR from a site I'd pulled down with Lando. The solution was to make the PR from a clean checkout of the repo from GitHub without basing it off a copy of the site/repo pulled down first through Lando/DDEV by way of Pantheon Git.

Do you have any insights?

**Review by mlsamuelson — CHANGES_REQUESTED** [review]

File: web/themes/custom/webdir/package-lock.json (hunk 2 above)
Comment (mlsamuelson) [reply web/themes/custom/webdir/package-lock.json:4]: From conversation. Lock file may have been out of date from the start due to copy-paste theme creation.

File: composer.json (hunk 3 above)
Comment (mlsamuelson) [reply composer.json:157]: Added based on interactive questions during `composer install`.

File: build-metadata.json (hunk 4 above)
Comment (tbutterf) [reply build-metadata.json:1]: I'll try doing a clean checkout and see what happens.

**Review by mlsamuelson — APPROVED** [review]
Thank you for the cleanup and answering my questions in Zoom.

Builds successfully.

I'm going to let the build-metadata.json and dev-master files in. They appear to be innocuous CI artifacts.

**Pull request merged**